filebeat/tests/load/filebeat.yml

```diff
@@ -15,7 +15,7 @@ filebeat:
   spool_size: 4096
   idle_timeout: 5s
   registry_file: registry
-
```

Comment:
This just triggered an idea on my end. It would be really great to have a system test that checks if the permission of the registry file is actually changed in case this is set differently. Can you add one?

Reply:
Alright.

Reply:
Will add tests for 0644, 0600 and 0664. IMO, that should suffice.

Reply:
Great to hear. Please ping us when you pushed a new commit. This will also show if it causes any problems on windows.